src/main/java/by/sanko/repo/Tweet.java

```diff
@@ -0,0 +1,82 @@
+package by.sanko.repo;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class Tweet {
+    int id;
+    String description;
+    Date createdAt;
+    String author;
+    String photoLink;
+    String photoFlag;
+    String photoRank;
+    String nation;
+    int rank;
+    int like;
+    ArrayList<String> hashTags;
+
+    public int getId() {
+        return id;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public Date getCreatedAt() {
+        return createdAt;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public String getPhotoLink() {
+        return photoLink;
+    }
+
+    public String getPhotoFlag() {
+        return photoFlag;
+    }
+
+    public String getPhotoRank() {
+        return photoRank;
+    }
+
+    public String getNation() {
+        return nation;
+    }
+
+    public int getRank() {
+        return rank;
+    }
+    public int getLike() {
+        return like;
+    }
+    public ArrayList<String>getTags() {
+        return hashTags;
+    }
+    public Tweet(int id, String description, Date createdAt, String author, String photoLink, String photoFlag, String photoRank, String nation, int rank,int likes,ArrayList<String> kk) {
```

Comment:
Please never declare a constructor with more that 3 or 4 arguments. Bad coding practice here. Use "builder" pattern, or use setters

Reply:
Should I change it?